docs/MCP.md

```
@@ -185,13 +187,13 @@ Once installed, `@gno` works anywhere in Raycast AI:
 | **AI Chat**     | Search "AI Chat" or assign hotkey (e.g., ⌥J) | Extended research, conversation history |
 | **AI Commands** | Custom commands with `@gno`                  | Repeatable workflows                    |
 
-**Quick AI** appears as a floating window above your apps — ideal for quick lookups:
+**Quick AI** appears as a floating window above your apps, ideal for quick lookups:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)